### ISSUE TITLE
URGENT!!! FBEE crashes on Windows Server 2003

### ISSUE THREAD

**Issue by ghost** [EnterpriseQualityCoding/FizzBuzzEnterpriseEdition#481]
The new version of the application crashes on our production servers running Windows Server 2003. A Windows upgrade is out of question because other production relevant software does not run on newer versions of Windows Server.
We expect a fix as soon as possible within 6 hours, as we are paying for the FBEE Ultra Platinum Business Support Plan for all of our servers.
Our production lines are standing still without FBEE!!!!!

**Comment by Tylersuard** [comment]
Sorry, but your support plan does not include support.  Only the Ultra Platinum Business DELUXE support plan includes support.  Please upgrade to take advantage of all our support features, such as waiting on hold and being told the problem is on your end.
